table.csv

```diff
@@ -429,3 +429,8 @@ holochain-key-v0,               holochain,      0x947124,       Holochain v0 pub
 holochain-key-v1,               holochain,      0x957124,       Holochain v1 public key + 8 R-S (63 x Base-32)
 holochain-sig-v0,               holochain,      0xa27124,       Holochain v0 signature  + 8 R-S (63 x Base-32)
 holochain-sig-v1,               holochain,      0xa37124,       Holochain v1 signature  + 8 R-S (63 x Base-32)
+fil-piece-unsealed,             serialization,  0xfi01,         Filecoin piece, raw data (CID = Piece Commitment)
```

Comment:
What's the format of this data? CAR?

Reply:
Should this be IPLD?

Reply:
It's neither. The CID here is the root of the merkletree constructed from the bytes that make up the CAR.